packages/blocks/src/paragraph-block/paragraph-block.ts

```diff
@@ -7,10 +7,14 @@ import {
 import { DisposableGroup, matchFlavours } from '@blocksuite/global/utils';
 import { BlockElement } from '@blocksuite/lit';
 import type { BaseBlockModel } from '@blocksuite/store';
-import { css, html } from 'lit';
+import { css, html, unsafeCSS } from 'lit';
 import { customElement, query, state } from 'lit/decorators.js';
 import { styleMap } from 'lit/directives/style-map.js';
 
+import {
+  quoteCSSContent,
+  quotePseudoElementContent,
+} from '../__internal__/content-parser/file-exporter/exporter-style.js';
```

Comment:
Thank you for your response. I appreciate your efforts to improve the CSS code. However, I think it would be better to use a common style for quotes rather than copying part of the CSS code. This can enhance the reusability of the CSS code and reduce the probability of different appearances between different modes (e.g., editor, exported HTML, markdown).

Actually, the "affine-quote-color" was defined several times with different values (I will put all the defines in the end). In this case, it would be best to consolidate these definitions into a single, common style for quotes. This will help to ensure consistency in the appearance of quotes across different modes.

[quote color 1](https://github.com/toeverything/blocksuite/blob/32e6e21dffcc4ccb9eb07eb987859295730345d8/packages/blocks/src/__internal__/content-parser/file-exporter/exporter-style.ts#L64)
[quote color 2](https://github.com/toeverything/blocksuite/blob/079b692347cde2ef71432815f8d016c025319d88/packages/docs/.vitepress/theme/style.css#L166)
[quote color 3](https://github.com/toeverything/design/blob/e6dbd5df9f7fd62b4b5f6cba26927ceb1c16d5dd/packages/theme/src/index.ts#L140)

Reply:
The FileExporter will be separated from the blocks package. So it would be better to not further adding the dependency here. @zuoxiaodong0815 will do some refactoring here to make sure this matches our plan here.